superset-frontend/packages/generator-superset/generators/plugin-chart/index.js

```diff
@@ -20,7 +20,14 @@
 /* eslint-disable sort-keys */
 
 const Generator = require('yeoman-generator');
-const _ = require('lodash');
```

Comment:
Can't we do `const { kebabCase, startCase, camelCase, upperFirst } = require('lodash');` instead?

Reply:
Just saw this after merging..... but... yes I suppose we can. I'll open a smaller/less contentious PR on some rainy day